examples/dicklang/parse_test.go

```diff
@@ -0,0 +1,44 @@
+// This is an example for the github.com/romshark/llparser library
+package main
+
+import (
+	"os"
+	"testing"
+
+	parser "github.com/romshark/llparser"
+	_ "github.com/romshark/llparser/misc"
+)
+
+// Set printParseTree to true to print the parse-tree
+// instead of the model
+var printParseTree = false
+
+// Change the source code however you like
+var src = `
```

Comment:
Avoid package-global variables, move this string to the actual test function instead.